10_install_server/00_install_intro.txt

```diff
@@ -25,7 +25,7 @@ Contains most of your server information, LDAP database, etc..
 Slower growth over time.
 
 /var/log/rudder:: 
-Reports logs can easily grow to 1.5GB per day.
+Reports logs can easily grow to 1.5GB per day in full compliance mode (much less in changes only mode).
```

Comment:
This is a random number. There is never any call for random numbers. Except maybe to confuse people.

We have actual metrics of the size here, please use them (see Rudder training).